src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const robotReverse = {};
+
+  for (const key in robot) {
+    if (robotReverse[robot[key]] === undefined) {
```

Comment:
This check is mentioned as a bad example of code style (checklist #2)

Reply:
agree